### DISCUSSION TITLE
templ CLI --proxy option and hot reload

### DISCUSSION THREAD

**Discussion by eussam** [a-h/templ#376] [Q&A] [answered]
Hi, 
I think it would be interesting for clarification to add a concrete example to the documentation about how to use the --proxy option with the --cmd option to setup hot reload with templ CLI.
@a-h do you think it could be interesting and if so shall I open a PR ?

**Comment by eussam** [comment, marked as answer]
great, answered here https://github.com/a-h/templ/issues/373#issuecomment-1879360406
closing 👍

**Comment by a-h** [comment]
Sorry, I missed this discussion. I would have loved a PR too on it too! 😁
